ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml

```diff
@@ -33913,13 +33913,7 @@ periodics:
     ci-operator.openshift.io/variant: amd64-stable
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-stable-aws-ocm-osd-ccs-p2-f7
-  reporter_config:
-    slack:
-      channel: '#ocm-qe-prow-ci-jobs'
-      job_states_to_report:
-      - failure
-      report_template: Job {{.Spec.Job}} failed. <{{.Status.URL}}|View logs>
```

Comment:
We lost slack notification for this job ?

Reply:
Good catch, fixed it.